src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultDocumentSnapshot.cs

```diff
@@ -27,14 +27,19 @@ public DefaultDocumentSnapshot(ProjectSnapshot project, DocumentState state)
             State = state;
         }
 
-        public ProjectSnapshot Project { get; }
+        public DefaultProjectSnapshot Project { get; }
```

Comment:
This is just setting up for the follow up PR.

Reply:
Sure